doc/src/modules/solvers/solveset.rst

```diff
@@ -476,10 +476,13 @@ What is the plan for solve and solveset?
 
 There are still a few things ``solveset`` can't do, which ``solve`` can, such
 as solving nonlinear multivariate & LambertW type equations.  Hence, it's not
-yet a perfect replacement for ``solve``. As the algorithms in ``solveset``
-mature, ``solveset`` may be able to be used within ``solve`` to replace some of
+a perfect replacement for ``solve``. As the algorithms in ``solveset`` mature,
+``solveset`` may be able to be used within ``solve`` to replace some of
 its algorithms.
```

Comment:
```suggestion
a perfect replacement for ``solve``.
```

Reply:
I would suggest just removing that 2nd sentence. How it is used where in SymPy is of no concern to the user.